README.md

```diff
@@ -67,6 +67,36 @@ agent.set('set', {
 agent.set('set', 'dimension15', 'My Custom Dimension')
 ```
 
+#### Register Plugin ( in plugin code )
+
+```js
+class MyPlugin {
+  constructor (agent, opts) {
+    this.opts = opts || {msg: ''}
+  }
+
+  say () {
```

Comment:
static say ?

Reply:
oh...

Reply:
it is class member function. not static.